### DISCUSSION TITLE
Help with donation

### DISCUSSION THREAD

**Discussion by staffxo** [Chatterino/chatterino2#5067] [Q&A] [answered]
I donated 5 euros for a badge on Chatterino homies and forgot to indicate my nickname, how can I get a badge?
![изображение](https://github.com/Chatterino/chatterino2/assets/155724435/39a212b1-4f45-4b41-9360-7ffaa2c626ad)
![изображение](https://github.com/Chatterino/chatterino2/assets/155724435/4c82e2aa-0213-4281-a6ee-ccfb426df73c)

nickname STAFFXO

**Comment by fourtf** [comment, marked as answer]
This is the discussion board for "Chatterino" and not "Chatterino Homies" so I suggest you get in contact with them directly. Some links (Discord/GitHub) are on their website https://chatterinohomies.com/

**Comment by Mm2PL** [comment]
If only we could force projects to not be named in a way that's confusing with ours.
